roles/contrail-provision-dockers/tasks/main.yaml

```diff
@@ -0,0 +1,31 @@
+- name: Debug hostvars 
+  debug:
+    var: hostvars[inventory_hostname]
+    verbosity: 4
```

Comment:
This won't work in zuul - it never sets verbosity over 3 IIRC. Either remove verbosity, or the task.